demos/multi_channel/yolo_v3/README.md

```diff
@@ -0,0 +1,118 @@
+
+> **NOTES**:
+> * Running the demo requires using at least one web camera attached to your machine.
+> * By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+```sh
+cd <samples_build_folder>/intel64/Release
+./multi-channel-yolo-v3-demo -h
+
+multichannel_yolo_v3 [OPTION]
+Options:
+
+    -h                           Print a usage message.
+    -m "<path>"                  Required. Path to an .xml file with a trained yolo v3 or tiny yolo v3 model.
+      -l "<absolute_path>"       Required for MKLDNN (CPU)-targeted custom layers. Absolute path to a shared library with the kernels impl.
+          Or
+      -c "<absolute_path>"       Required for clDNN (GPU)-targeted custom kernels. Absolute path to the xml file with the kernels desc.
+    -d "<device>"                Optional. Specify the target device for Face Detection (CPU, GPU, FPGA, HDDL or MYRIAD). The demo will look for a suitable plugin for a specified device.
+    -nc                          Optional. Maximum number of processed camera inputs (web cams)
+    -bs                          Optional. Batch size for processing (the number of frames processed per infer request)
+    -nireq                       Optional. Number of infer requests
+    -n_iqs                       Optional. Frame queue size for input channels
+    -fps_sp                      Optional. FPS measurement sampling period. Duration between timepoints, msec
+    -n_sp                        Optional. Number of sampling periods
+    -pc                          Optional. Enables per-layer performance report.
+    -t                           Optional. Probability threshold for detections.
+    -no_show                     Optional. No show processed video.
+    -show_stats                  Optional. Enable statistics report
+    -duplicate_num               Optional. Enable and specify number of channel additionally copied from real sources
+    -real_input_fps              Optional. Disable input frames caching, for maximum throughput pipeline
+    -i                           Optional. Specify full path to input video files
+
+```
+
+To run the demo, you can use public pre-train model and follow [this](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_YOLO_From_Tensorflow.html) page convert it into IR model. 
```

Comment:
please change statement to : .. page for instruction of how to convert it to IR model.